### PR TITLE
Gui: hide the selective sync pop up by default

### PR DESCRIPTION
**Pull request by TheOneRing**
This fixes a strange flickering when the client is started with the gui visible.

### CODE REVIEW

File: changelog/unreleased/7925 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Change: The settings ui shows a promt for a prompt for a few seconds
```
Comment (dschmidt) [inline changelog/unreleased/7925:1]: I don't understand this sentence

**Comment by jnweiger** [comment]
cannot conirm fix. changelog message is unclear, reproducer is unclear. Sorry.

**Comment by TheOneRing** [comment]
The widget for selective sync, (sync folders bigger than) in the lower pane was visible when starting the application with the -s for a second until we had a defined state.
Flashing widgets are not cool

**Comment by jnweiger** [comment]
## confirmed in testpilotcloud version 2.7.0daily20201019
started with -s, connecting to demo.owncloud.org
while the setup wizard is shown, nothing flashes. 

(cannot reproduce)

**Pull request merged**